Dockerfile-default-cpu

```diff
@@ -17,6 +17,7 @@ RUN if [ "$TENSORFLOW_PIP" ]; then \
         else \
             pip install $TENSORFLOW_PIP; \
         fi; \
+      pip install -r /tmp/det_dockerfile_scripts/additional-requirements-tf.txt; \
```

Comment:
wrong indentation, this should match the `fi;`, otherwise it's just confusing to the reader.

Also... this file has some weird indentation going on from past authors.  Can you convert the three lines using tabs to use spaces instead?